data/main/components.json

```diff
@@ -34,6 +34,7 @@
         "qbank": "public/question/bank",
         "qbehaviour": "public/question/behaviour",
         "qformat": "public/question/format",
+        "quizaccess": "public/mod/quiz/access",
```

Comment:
Please remove this - it is wrong.

The new component is a subplugin of mod_quiz and not a global plugin type.
The content of this file is automatically pulled in.